### PR TITLE
csi: fix template render in logRotation

### PR DESCRIPTION
**Pull request by Madhu-1**
The templateParam needs to be updated with the right values before we render the template.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by parth-gr — APPROVED** [review]
lgtm

**Comment by parth-gr** [comment]
Testing looks good,

```
[rider@localhost Downloads]$ oc exec -it csi-rbdplugin-ntlrd sh
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
Defaulting container name to driver-registrar.
Use 'oc describe pod/csi-rbdplugin-ntlrd -n openshift-storage' to see all of the containers in this pod.
sh-5.1# 
sh-5.1# exit
exit
[rider@localhost Downloads]$ oc exec -it csi-rbdplugin-ntlrd -c log-collector sh
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
sh-5.1# 
sh-5.1# 
sh-5.1# cd /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log
log/              logrotate-config/ 
sh-5.1# cd /var/lib/rook/openshift-storage.rbd.csi.ceph.com/logrotate-config/
sh-5.1# cd node-plugin/    
sh-5.1# cat csi 
/var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/*.log {
    daily
    missingok
    maxsize 524M
    rotate 7
    compress
    copytruncate
    notifempty
}
sh-5.1# cd ..
sh-5.1# cd ..
sh-5.1# cd log
log/              logrotate-config/ 
sh-5.1# cd log/node-plugin/csi-
csi-addons.log     csi-rbdplugin.log  
sh-5.1# cd log/node-plugin     
sh-5.1# 
sh-5.1# 
sh-5.1# cat csi-rbdplugin.log 
Log file created at: 2024/07/17 06:45:31
Running on machine: ip-10-0-95-17
Binary: Built with gc go1.22.4 (Red Hat 1.22.4-2.el9) for linux/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I0717 06:45:31.333763  225642 cephcsi.go:191] Driver version: devel and Git version: 8158cc63c37300197b609083034b6fff72472521
I0717 06:45:31.343569  225642 cephcsi.go:268] Initial PID limit is set to -1
I0717 06:45:31.343620  225642 cephcsi.go:274] Reconfigured PID limit to -1 (max)
I0717 06:45:31.343813  225642 cephcsi.go:223] Starting driver type: rbd with name: openshift-storage.rbd.csi.ceph.com
I0717 06:45:31.356590  225642 mount_linux.go:282] Detected umount with safe 'not mounted' behavior
I0717 06:45:31.356711  225642 rbd_attach.go:243] nbd module loaded
I0717 06:45:31.356741  225642 rbd_attach.go:257] kernel version "5.14.0-427.26.1.el9_4.x86_64" supports cookie feature
W0717 06:45:31.356814  225642 rbd_attach.go:263] running rbd-nbd --help failed with error:an error (exec: "rbd-nbd": executable file not found in $PATH) occurred while running rbd-nbd args: [--help], stderr:
I0717 06:45:31.357128  225642 server.go:114] listening for CSI-Addons requests on address: &net.UnixAddr{Name:"/csi/csi-addons.sock", Net:"unix"}
I0717 06:45:31.357250  225642 server.go:117] Listening for connections on address: &net.UnixAddr{Name:"//csi/csi.sock", Net:"unix"}
I0717 06:45:31.411091  225642 utils.go:235] ID: 1 GRPC call: /identity.Identity/Probe
I0717 06:45:31.411920  225642 utils.go:236] ID: 1 GRPC request: {}
I0717 06:45:31.412052  225642 utils.go:242] ID: 1 GRPC response: {"ready":{"value":true}}
I0717 06:45:31.413193  225642 utils.go:235] ID: 2 GRPC call: /identity.Identity/GetIdentity
I0717 06:45:31.413254  225642 utils.go:236] ID: 2 GRPC request: {}
I0717 06:45:31.413340  225642 utils.go:242] ID: 2 GRPC response: {"name":"openshift-storage.rbd.csi.ceph.com","vendor_version":"devel"}
I0717 06:45:31.425003  225642 utils.go:235] ID: 3 GRPC call: /identity.Identity/GetCapabilities
I0717 06:45:31.425071  225642 utils.go:236] ID: 3 GRPC request: {}
I0717 06:45:31.425218  225642 utils.go:242] ID: 3 GRPC response: {"capabilities":[{"Type":{"Service":{"type":2}}},{"Type":{"ReclaimSpace":{"type":2}}}]}
I0717 06:45:31.432556  225642 utils.go:235] ID: 4 GRPC call: /identity.Identity/GetCapabilities
I0717 06:45:31.432580  225642 utils.go:236] ID: 4 GRPC request: {}
I0717 06:45:31.432640  225642 utils.go:242] ID: 4 GRPC response: {"capabilities":[{"Type":{"Service":{"type":2}}},{"Type":{"ReclaimSpace":{"type":2}}}]}
I0717 06:45:32.195080  225642 utils.go:235] ID: 5 GRPC call: /csi.v1.Identity/GetPluginInfo
I0717 06:45:32.195836  225642 utils.go:236] ID: 5 GRPC request: {}
I0717 06:45:32.195860  225642 identityserver-default.go:40] ID: 5 Using default GetPluginInfo
I0717 06:45:32.195930  225642 utils.go:242] ID: 5 GRPC response: {"name":"openshift-storage.rbd.csi.ceph.com","vendor_version":"devel"}
I0717 06:45:32.635482  225642 utils.go:235] ID: 6 GRPC call: /csi.v1.Node/NodeGetInfo
I0717 06:45:32.635559  225642 utils.go:236] ID: 6 GRPC request: {}
I0717 06:45:32.635574  225642 nodeserver-default.go:45] ID: 6 Using default NodeGetInfo
I0717 06:45:32.635655  225642 utils.go:242] ID: 6 GRPC response: {"accessible_topology":{},"node_id":"ip-10-0-95-17.ec2.internal"}
I0717 06:45:49.817823  225642 utils.go:235] ID: 7 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 06:45:49.817911  225642 utils.go:236] ID: 7 GRPC request: {}
I0717 06:45:49.818021  225642 utils.go:242] ID: 7 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 06:45:49.819228  225642 utils.go:235] ID: 8 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 06:45:49.819293  225642 utils.go:236] ID: 8 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 06:45:49.819459  225642 utils.go:242] ID: 8 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
I0717 06:47:08.420942  225642 utils.go:235] ID: 9 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 06:47:08.420972  225642 utils.go:236] ID: 9 GRPC request: {}
I0717 06:47:08.421042  225642 utils.go:242] ID: 9 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 06:47:08.421656  225642 utils.go:235] ID: 10 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 06:47:08.421686  225642 utils.go:236] ID: 10 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 06:47:08.421767  225642 utils.go:242] ID: 10 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
I0717 06:48:44.555827  225642 utils.go:235] ID: 11 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 06:48:44.555866  225642 utils.go:236] ID: 11 GRPC request: {}
I0717 06:48:44.555935  225642 utils.go:242] ID: 11 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 06:48:44.556624  225642 utils.go:235] ID: 12 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 06:48:44.556660  225642 utils.go:236] ID: 12 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 06:48:44.556744  225642 utils.go:242] ID: 12 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
I0717 06:49:58.394898  225642 utils.go:235] ID: 13 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 06:49:58.394940  225642 utils.go:236] ID: 13 GRPC request: {}
I0717 06:49:58.395031  225642 utils.go:242] ID: 13 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 06:49:58.395669  225642 utils.go:235] ID: 14 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 06:49:58.395704  225642 utils.go:236] ID: 14 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 06:49:58.395790  225642 utils.go:242] ID: 14 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
I0717 06:51:28.585311  225642 utils.go:235] ID: 15 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 06:51:28.585355  225642 utils.go:236] ID: 15 GRPC request: {}
I0717 06:51:28.585479  225642 utils.go:242] ID: 15 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 06:51:28.586102  225642 utils.go:235] ID: 16 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 06:51:28.586138  225642 utils.go:236] ID: 16 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 06:51:28.586219  225642 utils.go:242] ID: 16 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
I0717 06:53:13.730484  225642 utils.go:235] ID: 17 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 06:53:13.730520  225642 utils.go:236] ID: 17 GRPC request: {}
I0717 06:53:13.730587  225642 utils.go:242] ID: 17 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 06:53:13.731248  225642 utils.go:235] ID: 18 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 06:53:13.731283  225642 utils.go:236] ID: 18 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 06:53:13.731370  225642 utils.go:242] ID: 18 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
I0717 06:54:49.790509  225642 utils.go:235] ID: 19 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 06:54:49.790552  225642 utils.go:236] ID: 19 GRPC request: {}
I0717 06:54:49.790619  225642 utils.go:242] ID: 19 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 06:54:49.791315  225642 utils.go:235] ID: 20 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 06:54:49.791351  225642 utils.go:236] ID: 20 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 06:54:49.791456  225642 utils.go:242] ID: 20 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
sh-5.1# 
sh-5.1# 
sh-5.1# ls -lh
total 20K
-rw-r--r--. 1 root root 1.8K Jul 17 06:45 csi-addons.log
-rw-r--r--. 1 root root  13K Jul 17 06:57 csi-rbdplugin.log
sh-5.1# 
sh-5.1# 
sh-5.1# truncate -s 525M csi-rbdplugin.log 
sh-5.1# 
sh-5.1# ls -lh
total 20K
-rw-r--r--. 1 root root 1.8K Jul 17 06:45 csi-addons.log
-rw-r--r--. 1 root root 525M Jul 17 06:58 csi-rbdplugin.log
sh-5.1# 
sh-5.1# 
sh-5.1# exit
exit
[rider@localhost Downloads]$ oc logs  csi-rbdplugin-ntlrd -c log-collector 
Starting the csi-logrotate-sidecar
File creation container completed
reading config file /var/lib/rook/openshift-storage.rbd.csi.ceph.com/logrotate-config/node-plugin/csi
Creating stub state file: /var/lib/logrotate/logrotate.status
Reading state from file: /var/lib/logrotate/logrotate.status
Allocating hash table for state file, size 64 entries

Handling 1 logs

rotating pattern: /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/*.log  after 1 days (7 rotations)
empty log files are not rotated, log files >= 549453824 are rotated earlier, old logs are removed
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-addons.log
Creating new state
  Now: 2024-07-17 06:45
  Last rotated at 2024-07-17 06:00
  log does not need rotating (log has already been rotated)
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log
Creating new state
  Now: 2024-07-17 06:45
  Last rotated at 2024-07-17 06:00
  log does not need rotating (log has already been rotated)
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ oc logs  csi-rbdplugin-ntlrd -c log-collector 
Starting the csi-logrotate-sidecar
File creation container completed
reading config file /var/lib/rook/openshift-storage.rbd.csi.ceph.com/logrotate-config/node-plugin/csi
Creating stub state file: /var/lib/logrotate/logrotate.status
Reading state from file: /var/lib/logrotate/logrotate.status
Allocating hash table for state file, size 64 entries

Handling 1 logs

rotating pattern: /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/*.log  after 1 days (7 rotations)
empty log files are not rotated, log files >= 549453824 are rotated earlier, old logs are removed
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-addons.log
Creating new state
  Now: 2024-07-17 06:45
  Last rotated at 2024-07-17 06:00
  log does not need rotating (log has already been rotated)
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log
Creating new state
  Now: 2024-07-17 06:45
  Last rotated at 2024-07-17 06:00
  log does not need rotating (log has already been rotated)
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ oc logs  csi-rbdplugin-ntlrd -c log-collector 
Starting the csi-logrotate-sidecar
File creation container completed
reading config file /var/lib/rook/openshift-storage.rbd.csi.ceph.com/logrotate-config/node-plugin/csi
Creating stub state file: /var/lib/logrotate/logrotate.status
Reading state from file: /var/lib/logrotate/logrotate.status
Allocating hash table for state file, size 64 entries

Handling 1 logs

rotating pattern: /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/*.log  after 1 days (7 rotations)
empty log files are not rotated, log files >= 549453824 are rotated earlier, old logs are removed
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-addons.log
Creating new state
  Now: 2024-07-17 06:45
  Last rotated at 2024-07-17 06:00
  log does not need rotating (log has already been rotated)
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log
Creating new state
  Now: 2024-07-17 06:45
  Last rotated at 2024-07-17 06:00
  log does not need rotating (log has already been rotated)
reading config file /var/lib/rook/openshift-storage.rbd.csi.ceph.com/logrotate-config/node-plugin/csi
Reading state from file: /var/lib/logrotate/logrotate.status
Allocating hash table for state file, size 64 entries
Creating new state
Creating new state

Handling 1 logs

rotating pattern: /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/*.log  after 1 days (7 rotations)
empty log files are not rotated, log files >= 549453824 are rotated earlier, old logs are removed
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-addons.log
  Now: 2024-07-17 07:00
  Last rotated at 2024-07-17 06:00
  log does not need rotating (log has been rotated at 2024-07-17 06:00, which is less than a day ago)
considering log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log
  Now: 2024-07-17 07:00
  Last rotated at 2024-07-17 06:00
  log needs rotating
rotating log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log, log->rotateCount is 7
dateext suffix '-20240717'
glob pattern '-[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]'
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.7.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.8.gz (rotatecount 7, logstart 1, i 7), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.7.gz does not exist
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.6.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.7.gz (rotatecount 7, logstart 1, i 6), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.6.gz does not exist
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.5.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.6.gz (rotatecount 7, logstart 1, i 5), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.5.gz does not exist
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.4.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.5.gz (rotatecount 7, logstart 1, i 4), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.4.gz does not exist
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.3.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.4.gz (rotatecount 7, logstart 1, i 3), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.3.gz does not exist
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.2.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.3.gz (rotatecount 7, logstart 1, i 2), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.2.gz does not exist
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.1.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.2.gz (rotatecount 7, logstart 1, i 1), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.1.gz does not exist
renaming /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.0.gz to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.1.gz (rotatecount 7, logstart 1, i 0), 
old log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.0.gz does not exist
log /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.8.gz doesn't exist -- won't try to dispose of it
copying /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log to /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log.1
truncating /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-rbdplugin.log
compressing log with: /bin/gzip
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ oc exec -it csi-rbdplugin-ntlrd -c log-collector sh
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
sh-5.1# cd /var/lib/r
rook/      rpcbind/   rpm/       rpm-state/ 
sh-5.1# cd /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log
log/              logrotate-config/ 
sh-5.1# cd /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log
log/              logrotate-config/ 
sh-5.1# cd /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin/csi-
csi-addons.log          csi-rbdplugin.log       csi-rbdplugin.log.1.gz  
sh-5.1# cd /var/lib/rook/openshift-storage.rbd.csi.ceph.com/log/node-plugin     
sh-5.1# ls
csi-addons.log	csi-rbdplugin.log  csi-rbdplugin.log.1.gz
sh-5.1# 
sh-5.1# ls -lh
total 536K
-rw-r--r--. 1 root root 1.8K Jul 17 06:45 csi-addons.log
-rw-r--r--. 1 root root 1.1K Jul 17 07:00 csi-rbdplugin.log
-rw-r--r--. 1 root root 525K Jul 17 06:59 csi-rbdplugin.log.1.gz
sh-5.1# cat csi-rbdplugin.log
I0717 07:00:45.273227  225642 utils.go:235] ID: 27 GRPC call: /csi.v1.Node/NodeGetCapabilities
I0717 07:00:45.273261  225642 utils.go:236] ID: 27 GRPC request: {}
I0717 07:00:45.273326  225642 utils.go:242] ID: 27 GRPC response: {"capabilities":[{"Type":{"Rpc":{"type":1}}},{"Type":{"Rpc":{"type":2}}},{"Type":{"Rpc":{"type":3}}},{"Type":{"Rpc":{"type":5}}}]}
I0717 07:00:45.274009  225642 utils.go:235] ID: 28 GRPC call: /csi.v1.Node/NodeGetVolumeStats
I0717 07:00:45.274045  225642 utils.go:236] ID: 28 GRPC request: {"volume_id":"0001-0011-openshift-storage-0000000000000001-b30de239-1a89-4847-8e2b-63a40680e98e","volume_path":"/var/lib/kubelet/pods/647bc4aa-fa8c-4359-a82e-525944d2d2eb/volumes/kubernetes.io~csi/pvc-86e7d499-4ec3-47f4-8149-4cca45e526c9/mount"}
I0717 07:00:45.274128  225642 utils.go:242] ID: 28 GRPC response: {"usage":[{"available":41940094976,"total":41956900864,"unit":1,"used":28672},{"available":2621426,"total":2621440,"unit":2,"used":14}],"volume_condition":{"message":"volume is in a healthy condition"}}
sh-5.1# 
sh-5.1# exit
exit
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ 
[rider@localhost Downloads]$ cd 


```

File: pkg/operator/ceph/csi/spec.go [hunk 1]
```diff
@@ -361,23 +361,22 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 	tp.Param.MountCustomCephConf = CustomCSICephConfigExists
 
 	if EnableRBD {
+		tp.CsiComponentName = nodePlugin
```
Comment (parth-gr) [inline pkg/operator/ceph/csi/spec.go:364]: we should check if csi log rotation is enabled then only append these values? 		

if tp.CSILogRotation {

**Review by parth-gr — CHANGES_REQUESTED** [review]

File: pkg/operator/ceph/csi/spec.go (hunk 1 above)
Comment (Madhu-1) [reply pkg/operator/ceph/csi/spec.go:364]: IMO we dont need to it, those values will be rendered only if the `tp.CSILogRotation` is set.

Comment (parth-gr) [reply pkg/operator/ceph/csi/spec.go:364]: make sense

**Review by parth-gr — APPROVED** [review]
tested and validated

**Review by subhamkrai — APPROVED** [review]

**Pull request merged**